ci/README.md

```diff
@@ -1,8 +1,9 @@
 # Continuous Integration Setup
 
-The CI environment is maintained by the SAP Business Technology Platform on Cloud Foundry team for Routing and Networking.
+The CI environment is maintained by the SAP Business Technology Platform on Cloud Foundry [team for Routing and Networking](https://github.com/orgs/cloudfoundry/teams/wg-app-runtime-platform-networking-extensions-approvers).
+
+All secrets used in the pipeline and surrounding tools are kept in the team's Vault. Those secrets are only necessary for [modifying the pipeline](#updating-or-modifying-the-pipeline). For git integration, [haproxy-boshrelease pipeline](https://concourse.arp.cloudfoundry.org/teams/main/pipelines/haproxy-boshrelease) has public visibility.
```

Comment:
This means that SAP employees are still the only ones who can change the pipelines?

Reply:
Let's say vault provides convenience. Otherwise you can get credentials from individual places (e.g. if you need slack webook that's used for reporting build failures, probably you can get it from slack directly - but you need to have correct level of access). I think some people from cloudfoundry org have access to all the individual places and credentials that are needed.

Reply:
I think for the scope of this task we could leave it like that. It would be feasible to move those (community) secrets to some secret store on GCP, I guess?